app/src/main/java/fr/free/nrw/commons/explore/SearchActivity.java

```diff
@@ -214,21 +214,6 @@ public void onSearchImageClicked(int index) {
         forceInitBackButton();
     }
 
-    /**
-     * This method is called on Screen Rotation
-     */
-    @Override
-    protected void onResume() {
-        if (supportFragmentManager.getBackStackEntryCount()==1){
-            //FIXME: Temporary fix for screen rotation inside media details. If we don't call onBackPressed then fragment stack is increasing every time.
-            //FIXME: Similar issue like this https://github.com/commons-app/apps-android-commons/issues/894
-            // This is called on screen rotation when user is inside media details. Ideally it should show Media Details but since we are not saving the state now. We are throwing the user to search screen otherwise the app was crashing.
-            // 
-            onBackPressed();
-        }
-        super.onResume();
-    }
-
```

Comment:
@domdomegg look here are some fixme and an onBackPressed call which I had to remove for this issue

Reply:
If it all still works and there are no issues pressing back or rotating the device then that's okay.

Reply:
Yes I have checked there are no issues on pressing back or rotating the device

Reply:
After testing can confirm this is the case. Still kinda worried, as it looks like it should be there for a reason. @ujjwalagrawal17 I believe you initially added this in 046533050e16e447b2d8bc25c816e1fd53dc9933, do you think you could provide feedback on this? Thanks!

Also can we remove the duplicate of this in `CategoryImagesActivity.java` without breaking something? (Best done in a different PR probably)

Reply:
@domdomegg Umm, when I added this the app was crashing in case of rotation when we are in explore screen, not inside media details. Here is the PR that I sent https://github.com/commons-app/apps-android-commons/pull/1626. We need to make sure https://github.com/commons-app/apps-android-commons/issues/1579 doesn't come again. If it works fine we can merge.